packages/viewer/src/components/scene-tree-table-header/scene-tree-table-header.css

```diff
@@ -0,0 +1,12 @@
+:host {
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  font-size: 0.875rem;
+  height: 2rem;
```

Comment:
You should let the content of the element dictate its height. Otherwise you have scenarios where the content can overflow its container.